docs/gen_ref_pages.py

```diff
@@ -0,0 +1,33 @@
+"""Generate the code reference pages."""
+
+from pathlib import Path
+
+import mkdocs_gen_files
+
+nav = mkdocs_gen_files.Nav()
+
+for path in sorted(Path("starlite_jwt").rglob("*.py")):  #
+    module_path = Path("starlite_jwt").with_suffix("")
+    doc_path = Path("starlite_jwt").with_suffix(".md")
+    print(f"{doc_path = }")
```

Comment:
```suggestion
```

Reply:
that stdout is captured and writes the files. I've noqa'd